src/styles/_landing.scss

```diff
@@ -359,32 +435,42 @@
     }
     @include retina {
        @media (max-width: 600px){
-        background-image: url('/assets/images/1on1-1200px.jpg');
+        background: #0b0b0c url('/assets/images/1on1-1200px.jpg') no-repeat -80vw -4vw / auto 110vw;
       }
     }
     > div {
       max-width: 80vw;
       padding: 10vw 40vw 10vw 0;
       box-sizing: border-box;
       margin: 0 auto;
+      @media (max-width: 600px) {
+        padding: 110vw 8vw 10vw;
+      }
     }
     h2 {
       color: white;
       font-family: $neuzeit-heavy;
       font-size: 4.2vw;
       line-height: 130%;
-      &:after {
-        content: ' ';
-        background: $peta-yellow;
-        height: 2px;
-        bottom: 0;
-        width: 30px;
-        position: absolute;
-        left: 0;
-      }
+      @media (max-width: 600px) {
+        font-size: 10vw;
+      }
+      // &:after {
+      //   content: ' ';
```

Comment:
This adds the little line underneath the heading. Either use `<hr>` or this CSS. But don't keep it as comment please 👍